CREDITS.md

```diff
@@ -72,16 +80,21 @@ Contributors to the Firecracker repository:
 * Ram Sripracha <ramsri@amazon.com>
 * Rob Devereux <robdevereux92@gmail.com>
 * Robert Grimes <rmzgrimes@gmail.com>
+* Rodrigue Chakode <rodrigue.chakode@continental-corporation.com>
 * Rolf Neugebauer <neugebar@amazon.com>
 * Sam Jackson <sam@clique.app>
 * Sean Lavine <freewil@users.noreply.github.com>
 * Serban Iorga <seriorga@amazon.com>
+* Shen Jiale <shenjiale@baidu.com>
 * Sripracha <ramsri@amazon.com>
+* Tamio-Vesa Nakajima <tamiove@amazon.com>
+* Tamio-Vesa Nakajima <tnk@amazon.com>
```

Comment:
This is a duplicate entry. Please update `.mailmap` to merge the 2 aliases for Tamio and remove the duplicate from the contributors list.

Reply:
Fixed.